release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -53,6 +53,10 @@ The installer now gathers serial console logs from the bootstrap and control pla
 
 For more information, see xref:../installing/installing-troubleshooting.adoc#installation-bootstrap-gather_installing-troubleshooting[Troubleshooting installation issues].
 
+[id="ocp-4-12-consistent-adddress-for-ironicapi"]
+==== Consistent IP address for Ironic API in bare-metal installations without a provisoning network  
```

Comment:
Can you double check whether it should be `Ironic` or `ironic`? It seems to be a new term in our docs. But I see in Bare metal RNs refer to it as `ironic` API.

Reply:
It is a new term in the docs. It's mostly transparent to users, that's why it occasionally surfaces in RNs. OpenStack and other references to this open-source API all use Ironic, for example https://docs.openstack.org/api-ref/baremetal/, so I think Ironic is correct.

Reply:
ah ok. I got to see a smaller case here https://docs.openstack.org/releasenotes/ironic/yoga.html#relnotes-20-1-0-stable-yoga-bug-fixes hence wasn't sure. Thanks for clarifying!